.gitignore

```diff
@@ -6,3 +6,4 @@ default.etcd
 oauth/client.json
 oauth/user.json
 .idea
+example.eskip
```

Comment:
why gitignore example.eskip? Seems like it could be useful to have some examples in the repo